app/routes/polls/components/PollEditor.js

```diff
@@ -0,0 +1,195 @@
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+import styles from './PollEditor.css';
+
+type Props = {
+  pristine: boolean,
+  submitting: boolean,
+  editOrCreatePoll: Object => Promise<*>,
+  handleSubmit: Object => Promise<*>,
+  editing: boolean,
+  initialValues: Object,
+  pollId: number,
+  deletePoll: () => Promise<*>,
+  toggleEdit: () => void
+};
+
+type FieldArrayPropTypes = {
+  fields: fieldArrayFieldsPropTypes,
+  meta: fieldArrayMetaPropTypes
+};
+
+const renderOptions = ({
+  fields,
+  meta: { touched, error }
+}: FieldArrayPropTypes) => (
+  <div>
+    <ul className={styles.options} key="options">
+      {fields.map((option, i) => (
+        <li className={styles.optionField} key={i}>
+          <Field
+            name={`${option}.name`}
+            label={`Valg nr. ${i + 1}`}
+            placeholder={`Valg ${i + 1}`}
+            component={TextInput.Field}
+            required
+          />
+          <ConfirmModalWithParent
+            title="Slett valg"
+            message="Er du sikker på at du vil slette dette valget?"
+            onConfirm={() => Promise.resolve(fields.remove(i))}
+            closeOnConfirm
+            className={styles.deleteOption}
+          >
+            <Icon name="trash" className={styles.deleteOption} />
+          </ConfirmModalWithParent>
+        </li>
+      ))}
+    </ul>
+
+    <Link
+      key="addNew"
+      onClick={() => {
+        fields.push({});
+      }}
+      className={styles.addOption}
+    >
+      <Icon name="add-circle" size={25} /> Valg
+    </Link>
+  </div>
+);
+
+class EditPollForm extends Component<Props, *> {
+  render() {
+    const {
+      pristine,
+      submitting,
+      handleSubmit,
+      editing,
+      deletePoll
+    } = this.props;
+
+    return (
+      <Content>
+        {!editing && (
+          <NavigationTab title="Ny avstemning">
+            <NavigationLink to="/polls">Tilbake</NavigationLink>
+          </NavigationTab>
+        )}
+        <Form onSubmit={handleSubmit}>
+          <Field
+            name="title"
+            label="Spørsmål"
+            placeholder="Hva er din favorittrett?"
+            component={TextInput.Field}
+            required
+          />
+          <span />
+          <Field
+            name="description"
+            label="Beskrivelse"
+            placeholder="Mer info..."
+            component={TextArea.Field}
+          />
+          <Field
+            name="pinned"
+            label="Vis på forsiden"
+            component={CheckBox.Field}
+          />
+          <Field
+            name="tags"
+            label="Tags"
+            filter={['tags.tag']}
+            placeholder="Skriv inn tags"
+            component={SelectInput.AutocompleteField}
+            multi
+            tags
+            shouldKeyDownEventCreateNewOption={({
+              keyCode
+            }: {
+              keyCode: number
+            }) => keyCode === 32 || keyCode === 13}
```

Comment:
Hmm... Maybe move these constants to a separate file (or atleast give them a name)?.